code/modules/mob/living/carbon/xenomorph/castes/defender/abilities_defender.dm

```diff
@@ -135,6 +135,17 @@
 	if(!do_after(X, 0.5 SECONDS, FALSE, X, BUSY_ICON_GENERIC, extra_checks = CALLBACK(src, .proc/can_use_ability, A, FALSE, XACT_USE_BUSY)))
 		return fail_activate()
 
+	var/mob/living/carbon/xenomorph/defender/D = X
```

Comment:
![image](https://user-images.githubusercontent.com/25491240/126480377-d223d375-048f-4a00-b425-16ed44690e45.png)

Please surr, this really helps with readability

Reply:
EXACTLY